ui-v2/app/components/topology-metrics/series/index.js

```diff
@@ -169,46 +173,51 @@ export default Component.extend({
     cursor.attr('x', mouseX);
 
     let mouseTime = x.invert(mouseX);
-    var bisectTime = bisector(function(d) { return d.time; }).left;
+    var bisectTime = bisector(function(d) {
+      return d.time;
+    }).left;
     let tipIdx = bisectTime(data, mouseTime);
 
     tooltip
       // 22 px is the correction to align the arrow on the tool tip with
       // cursor.
-      .style("left",(mouseX-22)+"px")
-      .select(".sparkline-time")
-        .text(niceTimeWithSeconds(mouseTime));
+      .style('left', mouseX - 22 + 'px')
+      .select('.sparkline-time')
+      .text(niceTimeWithSeconds(mouseTime));
 
     tipVals.nodes().forEach((n, i) => {
-      let val = stackData[i][tipIdx][1] - stackData[i][tipIdx][0];
-      select(n)
-        .text(this.formatTooltip(keys[i], val));
+      let tipData = stackData[i][tipIdx];
+      if (!tipData) return;
+
+      let val = tipData[1] - tipData[0];
```

Comment:
This is needed to guard against the `[]` return from `reformatSeries`, without that early return though (letting it throw an error instead) I think `stackData` might've been entirely undefined...

Reply:
Doesn't the fix you made on line 64 prevent this ever happening. I think if there is no series data we should never get this far. I'll look again.